<?xml version="1.0" encoding="UTF-8"?>
<testng-results retried="1" ignored="0" total="6" passed="4" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2025-03-22T13:12:47 IST" name="Suite" finished-at="2025-03-22T13:13:28 IST" duration-ms="41298">
    <groups>
      <group name="Purchase">
        <method signature="SubmitOrder.submitOrder(java.util.HashMap)[pri:0, instance:RahulShettyAcademy.Tests.SubmitOrder@26df6e3a]" name="submitOrder" class="RahulShettyAcademy.Tests.SubmitOrder"/>
      </group> <!-- Purchase -->
      <group name="ErrorHandling">
        <method signature="ErrorValidationTest.LoginErrorValidation()[pri:0, instance:RahulShettyAcademy.Tests.ErrorValidationTest@7b50df34]" name="LoginErrorValidation" class="RahulShettyAcademy.Tests.ErrorValidationTest"/>
      </group> <!-- ErrorHandling -->
    </groups>
    <test started-at="2025-03-22T13:12:47 IST" name="ErrorValidationTest" finished-at="2025-03-22T13:13:23 IST" duration-ms="36600">
      <class name="RahulShettyAcademy.Tests.ErrorValidationTest">
        <test-method is-config="true" signature="launchApplication()[pri:0, instance:RahulShettyAcademy.Tests.ErrorValidationTest@7b50df34]" started-at="2025-03-22T13:12:47 IST" name="launchApplication" finished-at="2025-03-22T13:12:50 IST" duration-ms="3405" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- launchApplication -->
        <test-method retried="true" signature="LoginErrorValidation()[pri:0, instance:RahulShettyAcademy.Tests.ErrorValidationTest@7b50df34]" started-at="2025-03-22T13:12:50 IST" name="LoginErrorValidation" finished-at="2025-03-22T13:12:51 IST" duration-ms="1160" status="SKIP">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [Incorrect email or password.] but found [Incorrect email or passwor.]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [Incorrect email or password.] but found [Incorrect email or passwor.]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1590)
at org.testng.Assert.assertEqualsImpl(Assert.java:150)
at org.testng.Assert.assertEquals(Assert.java:132)
at org.testng.Assert.assertEquals(Assert.java:656)
at org.testng.Assert.assertEquals(Assert.java:666)
at RahulShettyAcademy.Tests.ErrorValidationTest.LoginErrorValidation(ErrorValidationTest.java:25)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:687)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:995)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
at org.testng.TestRunner.privateRun(TestRunner.java:741)
at org.testng.TestRunner.run(TestRunner.java:616)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:463)
at org.testng.internal.thread.ThreadUtil.lambda$execute$0(ThreadUtil.java:64)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
at java.base/java.lang.Thread.run(Thread.java:1583)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- LoginErrorValidation -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:RahulShettyAcademy.Tests.ErrorValidationTest@7b50df34]" started-at="2025-03-22T13:12:51 IST" name="tearDown" finished-at="2025-03-22T13:12:51 IST" duration-ms="197" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method is-config="true" signature="launchApplication()[pri:0, instance:RahulShettyAcademy.Tests.ErrorValidationTest@7b50df34]" started-at="2025-03-22T13:12:51 IST" name="launchApplication" finished-at="2025-03-22T13:12:54 IST" duration-ms="2903" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- launchApplication -->
        <test-method signature="LoginErrorValidation()[pri:0, instance:RahulShettyAcademy.Tests.ErrorValidationTest@7b50df34]" started-at="2025-03-22T13:12:54 IST" name="LoginErrorValidation" finished-at="2025-03-22T13:12:55 IST" duration-ms="936" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [Incorrect email or password.] but found [Incorrect email or passwor.]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [Incorrect email or password.] but found [Incorrect email or passwor.]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1590)
at org.testng.Assert.assertEqualsImpl(Assert.java:150)
at org.testng.Assert.assertEquals(Assert.java:132)
at org.testng.Assert.assertEquals(Assert.java:656)
at org.testng.Assert.assertEquals(Assert.java:666)
at RahulShettyAcademy.Tests.ErrorValidationTest.LoginErrorValidation(ErrorValidationTest.java:25)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:687)
at org.testng.internal.invokers.TestInvoker.retryFailed(TestInvoker.java:269)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:76)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:995)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
at org.testng.TestRunner.privateRun(TestRunner.java:741)
at org.testng.TestRunner.run(TestRunner.java:616)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:463)
at org.testng.internal.thread.ThreadUtil.lambda$execute$0(ThreadUtil.java:64)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
at java.base/java.lang.Thread.run(Thread.java:1583)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- LoginErrorValidation -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:RahulShettyAcademy.Tests.ErrorValidationTest@7b50df34]" started-at="2025-03-22T13:12:56 IST" name="tearDown" finished-at="2025-03-22T13:12:56 IST" duration-ms="162" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method is-config="true" signature="launchApplication()[pri:0, instance:RahulShettyAcademy.Tests.ErrorValidationTest@7b50df34]" started-at="2025-03-22T13:12:56 IST" name="launchApplication" finished-at="2025-03-22T13:12:59 IST" duration-ms="2726" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- launchApplication -->
        <test-method signature="ProductErrorValidation()[pri:0, instance:RahulShettyAcademy.Tests.ErrorValidationTest@7b50df34]" started-at="2025-03-22T13:12:59 IST" name="ProductErrorValidation" finished-at="2025-03-22T13:13:23 IST" duration-ms="24091" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- ProductErrorValidation -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:RahulShettyAcademy.Tests.ErrorValidationTest@7b50df34]" started-at="2025-03-22T13:13:23 IST" name="tearDown" finished-at="2025-03-22T13:13:23 IST" duration-ms="1" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- RahulShettyAcademy.Tests.ErrorValidationTest -->
    </test> <!-- ErrorValidationTest -->
    <test started-at="2025-03-22T13:12:47 IST" name="SubmitOrder" finished-at="2025-03-22T13:13:28 IST" duration-ms="41298">
      <class name="RahulShettyAcademy.Tests.SubmitOrder">
        <test-method is-config="true" signature="launchApplication()[pri:0, instance:RahulShettyAcademy.Tests.SubmitOrder@26df6e3a]" started-at="2025-03-22T13:12:47 IST" name="launchApplication" finished-at="2025-03-22T13:12:50 IST" duration-ms="3328" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- launchApplication -->
        <test-method signature="submitOrder(java.util.HashMap)[pri:0, instance:RahulShettyAcademy.Tests.SubmitOrder@26df6e3a]" started-at="2025-03-22T13:12:50 IST" name="submitOrder" data-provider="getData" finished-at="2025-03-22T13:13:05 IST" duration-ms="14993" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[{password=IloveRahul$1956, product=ZARA COAT 3, email=santosh@gmail.com}]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- submitOrder -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:RahulShettyAcademy.Tests.SubmitOrder@26df6e3a]" started-at="2025-03-22T13:13:05 IST" name="tearDown" finished-at="2025-03-22T13:13:05 IST" duration-ms="226" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method is-config="true" signature="launchApplication()[pri:0, instance:RahulShettyAcademy.Tests.SubmitOrder@26df6e3a]" started-at="2025-03-22T13:13:05 IST" name="launchApplication" finished-at="2025-03-22T13:13:08 IST" duration-ms="2674" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- launchApplication -->
        <test-method signature="submitOrder(java.util.HashMap)[pri:0, instance:RahulShettyAcademy.Tests.SubmitOrder@26df6e3a]" started-at="2025-03-22T13:13:08 IST" name="submitOrder" data-provider="getData" finished-at="2025-03-22T13:13:23 IST" duration-ms="15151" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[{password=IlovemyMom$1956, product=ADIDAS ORIGINAL, email=santoshsujit@gmail.com}]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- submitOrder -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:RahulShettyAcademy.Tests.SubmitOrder@26df6e3a]" started-at="2025-03-22T13:13:23 IST" name="tearDown" finished-at="2025-03-22T13:13:24 IST" duration-ms="1073" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method is-config="true" signature="launchApplication()[pri:0, instance:RahulShettyAcademy.Tests.SubmitOrder@26df6e3a]" started-at="2025-03-22T13:13:24 IST" name="launchApplication" finished-at="2025-03-22T13:13:27 IST" duration-ms="2929" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- launchApplication -->
        <test-method signature="OrderHistoryTest()[pri:0, instance:RahulShettyAcademy.Tests.SubmitOrder@26df6e3a]" started-at="2025-03-22T13:13:27 IST" name="OrderHistoryTest" finished-at="2025-03-22T13:13:28 IST" depends-on-methods="RahulShettyAcademy.Tests.SubmitOrder.submitOrder" duration-ms="573" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- OrderHistoryTest -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:RahulShettyAcademy.Tests.SubmitOrder@26df6e3a]" started-at="2025-03-22T13:13:28 IST" name="tearDown" finished-at="2025-03-22T13:13:28 IST" duration-ms="141" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- RahulShettyAcademy.Tests.SubmitOrder -->
    </test> <!-- SubmitOrder -->
  </suite> <!-- Suite -->
</testng-results>
